sensor/RatingSensor.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import websockets
+import json
+
+from sensor.CSVLoader import generator_from_csv
+
+PORT = 8765
+URL = "localhost"
+RATINGS_FILE_PATH = './sensor/Ratings.csv'
+
+async def broadcast_rating_data(websocket, path):
+    for row in generator_from_csv(RATINGS_FILE_PATH):
+        await websocket.send(json.dumps(row))
```

Comment:
What if the websocket disconnects for some reason?

Reply:
Noted, I'll handle it after finishing a basic pipeline.